Automattic/wp-calypso
# Launchpad: Enable flows to use the new task definition parser

Related to #86647

## Proposed Changes
The idea here is to enable only a small set of flows by 1 week and check if there is no regression, so we can remove deprecated code with the PR  #86960

Currently, the onboarding flows"setup/design-first" and "setup/free" are using the refactored code. 
This PR enables more flows to use the refactored code. |
```JSON
"launchpad/new-task-definition-parser/start-writing": true,
"launchpad/new-task-definition-parser/videopress": true,
"launchpad/new-task-definition-parser/build": true,
"launchpad/new-task-definition-parser/newsletter": true,
"launchpad/new-task-definition-parser/link-in-bio": true,
``` 

This subset of flows contain all task definitions we have,  if we don't have issues we can remove the feature flags, enabling the refactored code to all flows and removing the deprecated code with the PR #86960

Move context p1706199762307079-slack-C0Q664T29

## Testing Instructions
* Run `yarn feature-search launchpad/new-task-definition-parser`
* Check if all tasks are enabled in production. 

Example:
<img width="516" alt="image" src="https://github.com/Automattic/wp-calypso/assets/38718/a8847ba6-2417-4e97-9358-1d9151a01e62">


## Pre-merge Checklist
